src/borg/testsuite/archiver.py

```diff
@@ -1680,6 +1680,12 @@ def test_log_json(self):
         assert log_message['name'].startswith('borg.')
         assert isinstance(log_message['message'], str)
 
+    def test_common_options(self):
+        self.create_test_files()
+        self.cmd('init', '--encryption=repokey', self.repository_location)
+        log = self.cmd('--debug', 'create', self.repository_location + '::test', 'input')
+        assert 'security: read previous_location' in log
```

Comment:
ok, so this a quite high-level test, like "does it work in production".
maybe have a more argparse-level test also / instead?